frontend/src/components/Registration/Registration.js

```diff
@@ -0,0 +1,24 @@
+import $ from 'jquery';
+
+export default function createUser(newUser) {
+  console.log('createUser called ' + JSON.stringify(newUser));
+  if (newUser) {
+    console.log('calling register endpoint');
+    $.ajax({
+      url: '/api/users',
```

Comment:
I also tried with just the url below but that didn't work either. the button didn't seem to do anything
```suggestion
      url: '/',
```

Reply:
ah no the button "get started" doesnt have any functionality yet. or at least i didn't add it.  best way to test it for now is postman i guess :)